stable/external-dns/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.5.4
+version: 2.5.5
```

```suggestion
version: 2.6.0
```
Thanks for the contribution, according to [semver](http://semver.org/), it seems more appropriate to increase the version in a minor due to you are adding new functionality:

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

---

Updated